onnx/test/optimizer_test.py

```diff
@@ -13,10 +13,11 @@
 
 class TestOptimizer(unittest.TestCase):
 
-    def _optimized(self, graph, opts):
+    def _optimized(self, graph, opts, check=True):
```

Comment:
Make checking unhappy does not look good. Let's store the input names in the attributes then :-)